CHANGELOG.md

```diff
@@ -14,9 +14,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Logic Database
 
+- Added: New trick, Cross Bomb Launch.
+
+##### Burenia
+
+- Added: Crossing the gap in Underneath Drogyga with Cross Bomb Launch (Advanced).
+- Added: Reaching the Missile Tank Pickup in Main Hub Tower Top using Cross Bomb Launch (Intermediate).
+- Changed: Using Cross Bombs to reach the item next to the fan in Burenia Hub to Dairon now Requires Cross Bomb Launch (Beginner) instead of Movement (Beginner).
+
+##### Cataris
+
+- Added: Reaching the Pickup in EMMI Zone Item Tunnel using Cross Bomb Launch (Advanced).
+
+##### Dairon
+
+- Added: Getting across the right gap in Early Grapple Room with Cross Bomb Launch (Beginner).
```

Comment:
Does this _really_ need to be a trick? Surely it can just be base logic since it's just using cross bombs over a gap the same width as the usual crumble blocks.